nano/node/active_transactions.hpp

```diff
@@ -245,14 +260,33 @@ class active_transactions final
 		mi::ordered_non_unique<mi::tag<tag_uncemented>,
 			mi::member<nano::cementable_account, uint64_t, &nano::cementable_account::blocks_uncemented>,
 			std::greater<uint64_t>>>>;
+
+	boost::multi_index_container<nano::expired_optimistic_election_info,
+	boost::multi_index::indexed_by<
+		boost::multi_index::ordered_non_unique<boost::multi_index::tag<tag_expired_time>,
+			boost::multi_index::member<expired_optimistic_election_info, std::chrono::steady_clock::time_point, &expired_optimistic_election_info::expired_time>>,
+		boost::multi_index::hashed_unique<boost::multi_index::tag<tag_account>,
+			boost::multi_index::member<expired_optimistic_election_info, nano::account, &expired_optimistic_election_info::account>>>>
```

Comment:
Can use the alias `mi` here for `boost::multi_index`

Reply:
Done thanks :)